internal/test/test_tpm.go

```diff
@@ -40,6 +41,26 @@ func (s simulatedTpm) EventLog() ([]byte, error) {
 	return s.eventLog, nil
 }
 
+// SkipOnAlgorithm skips the test if the algorithm is not found in the TPM
+// capability.
+func SkipOnAlgorithm(tb testing.TB, tpm io.ReadWriteCloser, alg tpm2.Algorithm) {
```

Comment:
I like go-tpm's name `skipOnUnsupportedAlgorithm`, as it makes clear that unsupported algorithms on a TPM will not run. Thoughts?

Reply:
changed